src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx

```diff
@@ -28,11 +28,16 @@ const ReportingReportConfirm = p => (
             </li>
             <li>
               <span>Gas</span>
-              <span>0.0023 ETH (2.8%)</span>
+              <span>-- ETH</span>
```

Comment:
Why did you change this to `-- ETH`? Just to show that it's not hooked up yet?

Reply:
yes, to show it's not hooked up.